### PR TITLE
Update DacFx to 162.1.80-preview and Projects to 0.1.11-preview

### PR DESCRIPTION
**Pull request by zijchen**

### CODE REVIEW

File: Packages.props [hunk 1]
```diff
@@ -22,8 +22,8 @@
 
 		<PackageReference Update="Microsoft.Data.SqlClient.AlwaysEncrypted.AzureKeyVaultProvider" Version="1.1.1" />
 		<PackageReference Update="Microsoft.SqlServer.Management.SmoMetadataProvider" Version="170.12.0" />
-		<PackageReference Update="Microsoft.SqlServer.DacFx" Version="162.1.59-preview" />
-		<PackageReference Update="Microsoft.SqlServer.DacFx.Projects" Version="162.1.37-alpha" />
+		<PackageReference Update="Microsoft.SqlServer.DacFx" Version="162.1.80-preview" />
+		<PackageReference Update="Microsoft.SqlServer.DacFx.Projects" Version="162.1.80-preview" />
```
Comment (llali) [inline Packages.props:26]: I didn't push this version of projects to nuget.org. Actually not sure how ST is getting it when it's not pushed to nuget.org. please double check with Benjin

Comment (llali) [reply Packages.props:26]: oh I see you are updating it in the source code. you might not need to if you only need dacfx changes but again make sure you ask Benjin about it

**Review by llali — APPROVED** [review]

File: Packages.props (hunk 1 above)
Comment (zijchen) [reply Packages.props:26]: From the original DacFx PR, there was a comment saying SqlPlatforms changes require a vbump of both DacFx and Projects Nuget packages - @Benjin can you confirm?

Comment (Benjin) [reply Packages.props:26]: I'm about to upload a version of Projects to nuget.org that references this project, so then you can grab that!  I'll ping you once I've done that.

**Review by Benjin — APPROVED** [review]

**Pull request merged**